src/plugins/dev_tools/public/plugin.ts

```diff
@@ -59,15 +63,18 @@ export class DevToolsPlugin implements Plugin<DevToolsSetup, void> {
       euiIconType: 'devToolsApp',
       order: 9001,
       category: DEFAULT_APP_CATEGORIES.management,
-      mount: async (appMountContext, params) => {
-        if (!this.getSortedDevTools) {
```

Comment:
This looks like a typo, so it always evaluates true. If this logic were to be changed to invoke the method and respect a `false` return value, we'd just throw an error and Kibana would crash. Which seems like worse behavior than rendering Dev Tools with an incomplete list of apps. So I think removing this condition makes the most sense.

Reply:
Yeah it looks like this check should have been:

```
this.getSortedDevTools().length === 0
```

But I think your proposal makes more sense :+1: